forge-gui/res/cardsfolder/upcoming/freestrider_commando.txt

```diff
@@ -0,0 +1,9 @@
+Name:Freestrider Commando
+ManaCost:2 G
+Types:Creature Centaur Mercenary
+PT:3/3
+K:etbCounter:P1P1:2:CheckSVar$ ETB:CARDNAME enters the battlefield with two +1/+1 counters on it if it wasn't cast or no mana was spent to cast it.
+SVar:ETB:Count$Valid Card.Self+wasNotCast,Card.Self+CastSa Spell.ManaSpent EQ0
```

Comment:
This isn't working

Reply:
imo just Compare against `Count$CastTotalManaSpent`?